client/src/app/(dev)/style-guide/buttons.tsx

```diff
@@ -1,17 +1,18 @@
 import React from "react";
```

Comment:
Do not skip heading levels: always start from `<h1>`, followed by `<h2>` and so on
Use only one `<h1>` elements on one page
[source](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements)

If you want to use h6 styling, do variant="h6" as="the_appropriate_element"